Gemfile

```diff
@@ -0,0 +1,5 @@
+source "https://rubygems.org"
+
+gem 'sinatra', '~> 1.4.6'
```

Comment:
I know it may not resonate much now, but getting in the habit of providing version constraints in your Gemfile is going to save you so many troubles in the future. Kudos.